engines/petka/detection.cpp

```diff
@@ -26,8 +26,8 @@
 #include "petka/petka.h"
 
 static const PlainGameDescriptor petkaGames[] = {
-	{"petka1", "Red Comrades 1: Save the Galaxy"},
-	{"petka2", "Red Comrades 2: For the Great Justice"},
+	{"petka1", "Петька и Василий Иванович спасают галактику"},
```

Comment:
Umm, please revert the title changes. The titles are taken from the official English re-releases on Android.

Reply:
All right, done